pyproject.toml

```diff
@@ -10,6 +10,7 @@ django-extensions = "3.2.3"
 django-celery-results = "2.5.1"
 djangorestframework = "3.14.0"
 gunicorn = "20.1.0"
+lxml = "5.1.0"
```

Comment:
cmd-F is not finding any other instances of `lxml` in this PR. Is this meant to be part of this pull request?

Reply:
Not sure why it didn't come up for you, but:

https://github.com/DDMAL/cantus/blob/a678ae69ec88b4158f57363df0f5cf66be6e6993/app/public/cantusdata/helpers/mei_processing/mei_parser.py#L17